docs/document_loader.ipynb

```diff
@@ -321,15 +321,131 @@
    },
    "outputs": [],
    "source": [
-    "loader = AlloyDBLoader.create(\n",
+    "loader = await AlloyDBLoader.create(\n",
     "    engine,\n",
-    "    table_name=\"products\",\n",
+    "    table_name=TABLE_NAME,\n",
     "    content_columns=[\"product_name\", \"description\"],\n",
     "    format=\"YAML\",\n",
     ")\n",
     "docs = await loader.aload()\n",
     "print(docs)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create AlloyDBDocumentSaver"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_google_alloydb_pg import AlloyDBDocumentSaver\n",
+    "\n",
+    "# Creating a basic AlloyDBDocumentSaver object\n",
```

Comment:
Can we also show how to create a table using method: `init_document_table()`?

Reply:
Added it just after engine initialization